WordPress/Classes/Utility/WPMediaEditor.swift

```diff
@@ -6,11 +6,19 @@ import Gridicons
  Displays the Media Editor with our custom styles and tracking
  */
 class WPMediaEditor: MediaEditor {
+    var alreadyPublishedImage: Bool = false {
```

Comment:
I'm not entirely sure what the alreadyPublishedImage means, is this a flag that means that we're editing an existing image?

Perhaps adding a documentation comment would be helpful in this instance?

```swift
    var alreadyPublishedImage: Bool = false {
```

Reply:
What do you think about:

```swift
    var editingAlreadyPublishedImage: Bool = false {
```

I can also add documentation.

Reply:
That's better.

Reply:
Done in e3cf0be8d7a2c756aef243b350d790c5efbfdb67